test/snippet/core/type_list/list_traits_contains.cpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/core/type_list/traits.hpp>
+
+int main()
+{
+    using t = seqan3::type_list<int, float, bool, int>;
+
+    bool contains = seqan3::list_traits::contains<double, t>; // Is type double in type_list t?
+    if (!contains)
+        seqan3::debug_stream << "Double not found.\n";
+    contains = seqan3::list_traits::contains<float, t>; // Is type float in type_list t?
```

Comment:
newline before next example

Reply:
@h-2 Is it okay if I use debug_stream here, or should I change this to a static_assert as well?

Reply:
maybe for consitency use 
`static_assert(seqan3::list_traits::contains<double, t> == false);`